CuratedContent/CMakeTutorialAndReferenceResources.md

```diff
@@ -0,0 +1,87 @@
+# CMake Tutorial and Reference Resources
+
+<!--deck text start-->
+The CMake meta-build system is a large and powerful set of tools to create cross-platform builds from source for C, C++, Fortran and many other compiled programming languages.
+CMake continues to rapidly evolve and best practices can change from year to year (and even from release to release in some cases).
+Here we provide links and descriptions for the best tutorial and reference materials that exist for CMake to help stay on the forefront of effective CMake usage.
+<!--deck text end-->
+
+#### Contributed by [Roscoe A. Bartlett](https://github.com/bartlettroscoe)
+#### Publication date: ???
+
+Resource information | Details 
+:--- | :--- 
+Book title | [Professional CMake](https://crascit.com/professional-cmake/)
+Authors | Craig Scott
+Publication | Self published, new editions put out after each new CMake release
+Web links | https://crascit.com/professional-cmake/
+
+Resource information | Details 
+:--- | :--- 
+Web Page | [CMake Reference Documentation](https://cmake.org/cmake/help/latest/)
+Authors | Kitware Staff
+Web links | https://cmake.org/cmake/help/latest/ (and drop-down section of all CMake versions)
+
+Resource information | Details 
+:--- | :--- 
+Book title | [Mastering CMake](https://crascit.com/professional-cmake/)
+Authors | Kitware Staff
+Publication | Year 2022, Self published, currently not versioned
+Web links | https://crascit.com/professional-cmake/
+
+The CMake meta-build tool and ecosystem (which includes the tools CTest and CPack with each CMake installation) continues to evolve and improve fairly rapidly from release to release.
```

Comment:
Does this expository text apply to the one item *just above*, to all of the items above or to the item coming next? Greater verticle spacing or a horizontal linke (in GFM thats a sep. line with just `---`) might help to make this clear

Reply:
All of the above.  So you suggest breaking out the intro with  `---`?  I will try that and see how it looks.

Reply:
Addressed in https://github.com/betterscientificsoftware/bssw.io/pull/1276/commits/af525be727200d27e6db242b72b4c1fb1ffe61b3

Reply:
Addressed in https://github.com/betterscientificsoftware/bssw.io/pull/1276/commits/af525be727200d27e6db242b72b4c1fb1ffe61b3